examples/azuread/sample_jupyter_config.py

```diff
@@ -9,3 +9,8 @@
 c.AzureAdOAuthenticator.oauth_callback_url = 'http://{your-domain}/hub/oauth_callback'
 c.AzureAdOAuthenticator.client_id = '{AAD-APP-CLIENT-ID}'
 c.AzureAdOAuthenticator.client_secret = '{AAD-APP-CLIENT-SECRET}'
+
+# if the user's name returned by Azure isn't acceptable to JupyterHub then we can use an alternate field,
+# uncomment the line below to use 'unique_name' rather than the default 'name'. Consult the Azure
+# documentation for other field names.
+#c.AzureAdOAuthenticator.username_claim = 'unique_name'
```

Comment:
For me the `name` returned by AD is bounced by JH so I had to use another value. This field was already present in the class, setting its value to `unique_name` solved my problem. I added this to the example code for others to more easily find than me.

See: https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/auth.py#L185